node-fs-writefile/note-to-self.js

```diff
@@ -0,0 +1,6 @@
+const data = new Uint8Array(Buffer.from(process.argv[2] + '\n'));
```

Comment:
While this works, it is a bit overkill, you can write a regular string to a file.